commands/wiki.js

```diff
@@ -24,13 +24,24 @@ files.forEach((file) => {
 		const fileName = path.basename(file, '.md');
 		const markdownContent = fs.readFileSync(filePath, 'utf-8');
 		const tokens = marked.lexer(markdownContent);
-		const headings = tokens.filter(token => token.type === 'heading' && token.depth === 1).map(token => token.text);
 
 		subcommands[reduce(fileName)] = {
 			file: fileName,
-			queries: headings,
+			queries: [],
 		};
 
+		tokens.forEach(token => {
+			if (token.type === 'heading' && token.depth === 1) {
+				subcommands[reduce(fileName)].queries.push([token.text, []]);
+			}
+			else if (token.type === 'heading' && token.depth === 2) {
+				const previoush1 = subcommands[reduce(fileName)].queries[subcommands[reduce(fileName)].queries.length - 1];
```

Comment:
i had to write it out to understand why you named it `previoush1` but looks good haha